src/com/liberty_amls/SettingsHandler.java

```diff
@@ -223,6 +223,17 @@ public void parseSettings() {
             if (settingsContainer.pushOSDAfterFrames < 0)
                 exitWithError("Wrong \"push OSD after frames\" number");
 
+            // Radius of a current planet that the project operates on
+            settingsContainer.planetRadius = jsonSettings.get("planet_radius").getAsInt();
+            if (settingsContainer.planetRadius <= 0)
+                exitWithError("Invalid planet radius");
+
+            // Distance in which the drone considered far enough to use K = 1
+            settingsContainer.notAcceptableDistance = jsonSettings.get("not_acceptable_distance").getAsInt();
+
+            // Debug option for whether to allow prediction of GPS coordinates or not
+            settingsContainer.allowPrediction = jsonSettings.get("allow_prediction").getAsBoolean();

```

Comment:
Please rename `allowPrediction` to `allowGPSPrediction` and `allow_gps_prediction`. Because it is not clear what kind of prediction we are talking about here

Reply:
Also, the style must match "landing_allowed"